package.json

```diff
@@ -469,6 +474,11 @@
           "when": "editorLangId =~ /^objectscript/ && vscode-objectscript.connectActive",
           "group": "objectscript@3"
         },
+        {
```

Comment:
My preference is for this command **not** to be added to the editor/context menu. That menu is already long, and users can't yet customize it. For people using a personal server namespace the command should be rarely needed. It's in Command Palette and a user can assign it a keyboard shortcut if they wish.